utils/update_translation_stats.py

```diff
@@ -85,68 +82,63 @@ def generate_translation_stats(po_dir, output_file):
                 break
             header = header + ' ' + line
         header_entries = header.split(',')
-        column_counter = 0
-        total_column = 0
-        translated_column = 0
-        while column_counter < len(header_entries):
-            if header_entries[column_counter].strip() == 'Total Source Words':
+        total_column = None
+        translated_column = None
+        for column_counter, cell in enumerate(header_entries):
+            if cell.strip() == 'Total Source Words':
                 total_column = column_counter
-            elif header_entries[column_counter].strip() == 'Translated Source Words':
+            elif cell.strip() == 'Translated Source Words':
                 translated_column = column_counter
-            column_counter = column_counter + 1
+        if not total_column or not translated_column:
+            for_column = 'total_column' if not total_column else translated_column
```

Comment:
I think you mean:

```suggestion
            for_column = 'total_column' if not total_column else 'translated_column'
```

But what if neither was found?

Reply:
Oops, thanks for catching

> But what if neither was found?

Then only "total_column" is reported.

Reply:
I know :) but I don't think that's right. Either print them separately as needed, or add one more condition for `'total_column and translated_column'`. But not as an inline `if`, please.

Also, this now reports "column for total_column not found", which is not nice, and `total_column` is internal to the script, so it doesn't belong in this message. The actual column names needed by the script should be reported instead.